package/package.json

```
@@ -45,7 +45,8 @@
     "release-next": "FILTER_PATH='package' TAG_FORMAT='v${version}' node ../release/next",
     "prepublishOnly": "echo 'running prepublishOnly for core package' && bash ./bin/before-tag.sh && bash ./bin/release.sh && echo 'finished running prepublishOnly for core package'",
     "prepack": " cp ../README.md .",
-    "postpack": "rm README.md"
+    "postpack": "rm README.md",
+    "update-emoji": "cd src/emoji-data && node compile.js"
```

```suggestion
    "update-emoji": "node compile.js"
```

can we make this possible?

---

No, this will create the file at the root

---

@khushal87 you could find the file directory inside the node script and create the file using that

---

Fixed